cmd/template/framework/files/main/main.go.tmpl

```diff
@@ -2,14 +2,26 @@ package main
 
 import (
 	"fmt"
+	"log/slog"
```

Comment:
There is also a fiber main.go, so the logger init needs to be included in this template as well